src/Docker.DotNet/Models/ContainerSpec.Generated.cs

```diff
@@ -78,7 +78,13 @@ public class ContainerSpec // (swarm.ContainerSpec)
         [DataMember(Name = "Sysctls", EmitDefaultValue = false)]
         public IDictionary<string, string> Sysctls { get; set; }
 
-        [DataMember(Name = "Capabilities", EmitDefaultValue = false)]
-        public IList<string> Capabilities { get; set; }
+        [DataMember(Name = "CapabilityAdd", EmitDefaultValue = false)]
```

Comment:
This must have been a Docker breaking change?

Reply:
Indeed 
https://github.com/moby/moby/commit/24f173a003727611aa482a55b812e0e39c67be65#diff-fe6d8227bdd6e327416738e82622b2b968a41c5451babe5588051df33f189c07

Reply:
Thanks